mobile/src/home-view.js

```diff
@@ -112,10 +99,11 @@ class HomeView extends PureComponent {
                 await this.props.fbc.signinAdmin()
                 console.log('Re-logged in as admin')
                 this.setState({ isAdmin: true })
-                getAdmin()
+                this.getAdmin()
               } else {
                 this.setState({ isAdminView: false })
               }
+              this.wireListeners()
```

Comment:
where does `wireListeners` happen if there is not an admin token?

Reply:
Doesn't this.props.fbc.database.private.adminableUserRef('adminToken').once('value', async data => return something regardless if their is data (admin) or not?